src/hexo/themes/sonarwhal/helper/index.js

```diff
@@ -269,6 +269,9 @@ module.exports = function () {
         isNotSectionIndexPage: (page) => {
             return !isSectionIndexPage(page);
         },
+        isPass: (status) => {
+            return status === jobStatus.finished;
```

Comment:
with this function, you are showing the pass icon, even if there is rules that doesn't pass.

IMO, we should show the pass icon if the scanner reports no errors.

@molant, @alrra thoughts?

Reply:
Yes, that's the idea:

* in progress
* finished OK
* finished errors

Reply:
@molant Just to be sure:
* scan finished, has errors reported - error favicon
* scan finished, all rules pass - pass favicon
* scan in progress - pending favicon.

What if there are failed scans (like when we scan github.com). We use the error favicon in this case as well?